rktl_autonomy/nodes/rocket_league_agent

```diff
@@ -10,7 +10,17 @@ from rktl_autonomy import RocketLeagueInterface
 from stable_baselines3 import PPO
 from os.path import expanduser
 import rospy
-
+"""
+overview:
+Load the weights for the specify model (PPO in this case)
+Create instance of the rocket_league_interface
+Reset the model for the rocket_league_interface
+Crete a PPO model based off the weights
+Infinite loop: 
+- predict the action based of the model
+- get the observation
+- step the action for the specific step based on the action 
+"""
```

Comment:
Personally, I think that by providing these notes, a person will be able to start working on the project faster (getting caught up and understanding the repo better). They have helped me out a lot (probably bc they are personal notes lol). 
Is it possible to remove them from the final commit and then add them in? 
I can also write more straightforward documentation as in the one that you recommend on lines 1-7 or those present in other functions that were not written by me. I just really think that there should be a blurb of comments to describe key functionality in the code and for each function.

Reply:
Wiki would be a good place for onboarding notes.

For code comments, you would want to split these paragraphs into: file summaries (top of file), function summaries (top of function), deobfuscation comments (inline). You want to avoid stating the obvious, for sake of clarity, especially since python is **usually** very readable. Sometimes you may opt for rewriting code, as opposed to adding comments.

Reply:
I reworked all of my documentation and code comments for the repo. Can you take another look?